src/app/src/components/StackedBarChart.js

```diff
@@ -54,7 +54,15 @@ const StackedBarChart = ({ items }) => {
                     maxValue={100}
                     groupMode='stacked'
                     layout='horizontal'
-                    colors={{ scheme: 'dark2' }}
+                    colors={item => {
+                        if (item.id === 'Strongly Disagree/Disagree') {
+                            return 'rgb(54, 17, 52)';
+                        } else if (item.id === 'Neutral') {
+                            return 'rgb(243, 164, 142)';
+                        } else {
```

Comment:
The `else` here and on other charts is working, but does leave open the possibility that changes to the data might start rendering in unexpected ways if any of the labels are changed. In particular, the "agree/disagree" style has 5 classes in the original survey that were compressed to 3 for the data we ingest. This may change in the future, in which case we'd start rendering different color combos without a warning. In this case, I think being explicit is safer (even if it doesn't change the functionality). I'd add another `else if` for each conditional that is currently implied by the `else`, and then have the else print a console.warning that it received an unexpected item, and then return a more neutral color, like gray. I think that would very quickly help us debug the problem if the data changes in unexpected ways in the future.

Reply:
Good catch. I've updated to send a warning and use a shade of grey from the mockups for `else`.